.travis.yml

```diff
@@ -1,13 +1,36 @@
+# Tasks are commented out purposfully. Will be uncommented for webpack-defaults application
 sudo: false
 language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4"
+branches:
+  only:
+    - master
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      node_js: '6'
+      env: JOB_PART=test
+    - os: linux
+      node_js: '4.3'
+      env: JOB_PART=test
+    - os: linux
+      node_js: '7'
+      env: JOB_PART=test
+    # - os: linux
+    #   node_js: '7'
+    #   env: WEBPACK_VERSION="2.2.0" BITHOUND_CHECK=true JOB_PART=lint
+    # - os: linux
+    #   node_js: '7'
+    #   env: WEBPACK_VERSION="2.2.0" JOB_PART=coverage
+before_install:
+  - nvm --version
+  - node --version
+before_script:
+  # - if [ "$WEBPACK_VERSION" ]; then yarn add webpack@^$WEBPACK_VERSION; fi
+  # - if [ "$BITHOUND_CHECK" ]; then npm install -g bithound; bithound check git@github.com:$TRAVIS_REPO_SLUG.git; fi
 script:
-  - "npm test"
-  # test with npm@3
-  - "rm -rf node_modules && npm install npm@3"
-  - "node_modules/.bin/npm install"
-  - "node_modules/.bin/npm test"
+  - yarn run travis:$JOB_PART
```

Comment:
The loader-utils update forces node 4 as a minimum supported version ( old travis file tests against 0.10.x & 0.12.x). Given I had to update it anyway, I grabbed the travis file that will be added as part of the pending `webpack-defaults` upgrade ( it's used in other libs already ).

So while it's not directly related, it is a side effect of the forced drop of the older node versions.